### PR TITLE
feat: add method for getting attribute policy collections 

### PR DESCRIPTION
**Pull request by cuadradek**
- Added new objects AttributePolicy and AttributePolicyCollection. They
will be used for evaluating users' rights on attributes. User has
rights to perform an action (READ/WRITE) on the attribute, if he
satisfies all policies in at least one of the attribute policy
collections with given action.
- Added new method getAttributePolicyCollections, it gets attribute
policy collections for given attribute.

### CODE REVIEW

File: perun-core/src/main/java/cz/metacentrum/perun/core/bl/AttributesManagerBl.java [hunk 1]
```diff
@@ -4620,6 +4621,16 @@ void mergeAttributesValues(PerunSession sess, Member member, List<Attribute> att
 	 */
 	void setAttributeRights(PerunSession sess, List<AttributeRights> rights);
 
+	/**
+	 * Gets attribute policy collections for an attribute definition with given id.
+	 *
+	 * @param sess perun session
+	 * @param attributeId id of the attribute definition
+	 * @return all policy collections of the attribute definition
+	 * @throws InternalErrorException
```
Comment (metodej) [inline perun-core/src/main/java/cz/metacentrum/perun/core/bl/AttributesManagerBl.java:4630]: I think there is no need for that in a javadoc, can we remove it?

File: perun-core/src/main/java/cz/metacentrum/perun/core/implApi/AttributesManagerImplApi.java [hunk 2]
```diff
@@ -2706,4 +2707,14 @@
...
 	 */
 	String escapeQueryParameters(String value);
+
+	/**
+	 * Gets attribute policy collections for an attribute definition with given id.
+	 *
+	 * @param sess perun session
+	 * @param attributeId id of the attribute definition
+	 * @return all policy collections of the attribute definition
+	 * @throws InternalErrorException
```
Comment (metodej) [inline perun-core/src/main/java/cz/metacentrum/perun/core/implApi/AttributesManagerImplApi.java:2717]: Same here.

**Review by metodej — APPROVED** [review]

File: perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java [hunk 3]
```diff
@@ -4923,6 +4968,20 @@ public void setAttributeRight(PerunSession sess, AttributeRights rights) {
 		}
 	}
 
+	@Override
+	public List<AttributePolicyCollection> getAttributePolicyCollections(PerunSession sess, final int attributeId) {
+		List<AttributePolicyCollection> attributePolicyCollections;
+		try {
+			attributePolicyCollections = jdbc.query("SELECT " + attributeRightSelectQuery + " FROM attributes_policies " +
```
Comment (Johaney-s) [inline perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java:4975]: I think `attributeRightSelectQuery` expects different tables than you provide

**Review by Johaney-s — CHANGES_REQUESTED** [review]
Looks good, just a few comments. Should we also add the new objects to `ParseRpcMethods`?

**Pull request merged**